Update csv

melinadewi opened the pull request:
Add Hari Sumpah Pemuda & Hari Pahlawan

melinadewi committed:
Update csv

khasburrahman commented:
Can we have the source? 
Thank you!

khasburrahman commented:
Found the source: 

[Hari sumpah pemuda](https://id.wikipedia.org/wiki/Sumpah_Pemuda)
> Sejak 1959, tanggal 28 Oktober ditetapkan sebagai Hari Sumpah Pemuda, yaitu hari nasional yang bukan hari libur yang ditetapkan oleh pemerintah Indonesia melalui Keppres No. 316 Tahun 1959 tanggal 16 Desember 1959 untuk memperingati peristiwa Sumpah Pemuda.

[Hari Pahlawan](https://id.wikipedia.org/wiki/Hari_Pahlawan_(Indonesia))
> Hari Pahlawan adalah hari nasional yang bukan hari libur yang ditetapkan oleh pemerintah Indonesia, dan diperingati pada tanggal 10 November setiap tahunnya di Indonesia.[1] Hari ini untuk memperingati Pertempuran Surabaya yang terjadi pada tahun 1945,[2] di mana para tentara dan milisi indonesia yang pro-kemerdekaan berperang melawan tentara Britania Raya dan Belanda yang merupakan bagian dari Revolusi Nasional Indonesia.[3] Hari nasional ini ditetapkan melalui Keppres No. 316 Tahun 1959 tanggal 16 Desember 1959.

khasburrahman commented on repository/ID/recurring_events/ID-recurring-events.csv:
```diff
@@ -3,4 +3,6 @@ XXXX-01-01,ID,T,XXXX-XXXX,Tahun Baru Masehi
 XXXX-05-01,ID,T,2014-XXXX,Hari Buruh
 XXXX-06-01,ID,T,2017-XXXX,Hari kelahiran Pancasila
 XXXX-08-17,ID,T,1945-XXXX,Proklamasi Kemerdekaan Republik Indonesia
+XXXX-10-28,ID,T,1928-XXXX,Hari Sumpah Pemuda
+XXXX-11-10,ID,T,1945-XXXX,Hari Pahlawan
```
```suggestion
XXXX-10-28,ID,T,1959-XXXX,Hari Sumpah Pemuda
XXXX-11-10,ID,T,1959-XXXX,Hari Pahlawan
```

khasburrahman committed:
Fix: update ID sumpah pemuda & pahlawan

Updated the starting year of holiday

Hari sumpah pemuda

Sejak 1959, tanggal 28 Oktober ditetapkan sebagai Hari Sumpah Pemuda, yaitu hari nasional yang bukan hari libur yang ditetapkan oleh pemerintah Indonesia melalui Keppres No. 316 Tahun 1959 tanggal 16 Desember 1959 untuk memperingati peristiwa Sumpah Pemuda.

Hari Pahlawan

Hari Pahlawan adalah hari nasional yang bukan hari libur yang ditetapkan oleh pemerintah Indonesia, dan diperingati pada tanggal 10 November setiap tahunnya di Indonesia.[1] Hari ini untuk memperingati Pertempuran Surabaya yang terjadi pada tahun 1945,[2] di mana para tentara dan milisi indonesia yang pro-kemerdekaan berperang melawan tentara Britania Raya dan Belanda yang merupakan bagian dari Revolusi Nasional Indonesia.[3] Hari nasional ini ditetapkan melalui Keppres No. 316 Tahun 1959 tanggal 16 Desember 1959.
